docs/pages/choose-an-edition/teleport-cloud/getting-started.mdx

```diff
@@ -1,90 +1,189 @@
+  instances that enable access to specific resources in your infrastructure.
 
-## Step 2/5 Access Web Console
+In this guide, you will register a server with your Teleport Cloud account so
+you can access it securely. Your server will connect to the Teleport Proxy
+Service and open a reverse tunnel that the Proxy Service can use to allow
+traffic to the server from authorized clients.
 
-Access Web Console
-![Access Web Console](../../../img/cloud/homepage-1.png)
+## Prerequisites
 
-Select Add Server and press COPY to copy the script command
-![Add Server Script](../../../img/cloud/addserver-2.png)
+- A Teleport Enterprise Cloud account. You can sign up for a free trial at the
+  following page:
 
-## Step 3/5 Install Teleport Agent on Server
+  https://goteleport.com/signup/
 
-Paste and run the script on a Linux Server:
+- Docker installed on your workstation. We will use a Docker container to
+  register a server with Teleport via the Teleport SSH Service. You can find
+  installation instructions on [Docker's
+  website](https://docs.docker.com/get-docker/).
 
-![Past Install Script](../../../img/cloud/runscript-3.png)
+  <Notice type="tip">
 
-Teleport Agent Installed
-![Completed Install](../../../img/cloud/completed-script-4.png)
+  Docker is only required for a local demo environment. If you want to get
+  started registering your servers in your infrastructure with Teleport, view
+  our documentation on how to [register servers and configure access to
+  them](../../server-access/introduction.mdx).
 
-## Step 4/5 Access Server
+  </Notice>
 
-Click **CLOSE**.  The server can now be accessed via SSH:
-![Access Server](../../../img/cloud/online-session-5.png)
+- The `tsh` client tool. Visit our [Installation Guide](../../installation.mdx)
+  for instructions. If you want to get started quickly, you can skip this for
+  now and access Teleport through your browser.
 
-Interact with your SSH session:
-![Open SSH](../../../img/cloud/online-ssh-5.png)
+## Step 1/4. Spin up your server
 
-Exit the session with `exit`. You can replay the session within Session Recordings:
-![Replay Session](../../../img/cloud/online-replay-5.png)
+Spin up a server on your workstation. In Step 2, we will register this server
+with your Teleport Enterprise Cloud tenant:
 
-## Step 5/5 Access from Command Line
+```code
+$ docker run -it ubuntu:22.10 /bin/bash
+```
+
+Make sure you have installed `curl` in your container:
+
+```code
+$ apt-get update
+$ apt-get install -y curl
+```
+
+Keep this shell open in your terminal for the next step.
+
+## Step 2/4. Install the Teleport SSH Service
+
+To install the Teleport SSH Service on your server, visit the Teleport Web UI at
+the address of your Teleport Enterprise Cloud tenant. For example, visit:
+
+https://mytenant.teleport.sh
+
+If this is your first time visiting the Web UI, you will see a prompt to add
+your first resource: 
+
+![Add your first
```

Comment:
The images may be compressed, but they are still huge in both file size and resolution for the web:

![image](https://user-images.githubusercontent.com/2349184/215551719-d3eb99f8-bffa-47f1-9d7c-6accb238c372.png)

Consider scaling them down to something closer to the maximum width of the `DocsPage_main` element to increase load time.

Reply:
I've added the `@2x` marker to the filenames (https://goteleport.com/docs/contributing/documentation/reference/#image-pixel-density-markers). In any case, it looks like NextJS is converting the PNG images to Webp and lazy-loading them at reasonable sizes, so load time shouldn't be an issue